ui/apps/ui/src/app/components/top-menu/top-menu.component.ts

```diff
@@ -39,8 +45,24 @@ export class TopMenuComponent implements OnInit {
   setPrefix(prefix: string) {
     localStorage.setItem('COLLECTIONS_PREFIX', prefix);
     this.selectedPrefix = prefix;
+    this.setCollection();
 
     // TODO: Make this an observable and reload only the affected components
+  }
+  async setCollection() {
+    const currentCollection: string = this._customRoute.collection() || '';
+    const availableCollections: string[] =
+      this.selectedPrefix == 'eu'
```

Comment:
its fine but nonetheless, I would reverse it:
`const availableCollections: string[] = this.selectedPrefix == 'pl' ? PL_CONTEXT_COLLECTIONS : EU_CONTEXT_COLLECTIONS`
"pl" is kind of a special narrower case, default should be the most generic, broad -> EU collections